cli/pkg/helm/helm.go

```diff
@@ -152,6 +153,7 @@ func (h *Helm) runHelm(args []string) error {
 					ServiceAccountName: serviceAccountName,
 				},
 			},
+			BackoffLimit: util.Int32Ptr(0),
```

Comment:
This is needed so that `airy create` or `airy upgrade` run only one job, without repeating. Easier to debug and to rerun if needed.